com.unity.ml-agents/Runtime/Inference/ApplierImpl.cs

```diff
@@ -85,99 +85,6 @@ public void Apply(TensorProxy tensorProxy, IList<int> actionIds, Dictionary<int,
         }
     }
 
-    internal class MaxActionOutputApplier : TensorApplier.IApplier
-    {
-        readonly ActionSpec m_ActionSpec;
-
-
-        public MaxActionOutputApplier(ActionSpec actionSpec, int seed, ITensorAllocator allocator)
-        {
-            m_ActionSpec = actionSpec;
-        }
-
-        public void Apply(TensorProxy tensorProxy, IList<int> actionIds, Dictionary<int, ActionBuffers> lastActions)
-        {
-            var agentIndex = 0;
-            var actionSpaceSize = tensorProxy.shape[tensorProxy.shape.Length - 1];
-
-            for (var i = 0; i < actionIds.Count; i++)
-            {
-                var agentId = actionIds[i];
-                if (lastActions.ContainsKey(agentId))
-                {
-                    var actionBuffer = lastActions[agentId];
-                    if (actionBuffer.IsEmpty())
-                    {
-                        actionBuffer = new ActionBuffers(m_ActionSpec);
-                        lastActions[agentId] = actionBuffer;
-                    }
-                    var discreteBuffer = actionBuffer.DiscreteActions;
-                    var maxIndex = 0;
-                    var maxValue = 0;
-                    for (var j = 0; j < actionSpaceSize; j++)
-                    {
-                        var value = (int)tensorProxy.data[agentIndex, j];
-                        if (value > maxValue)
-                        {
-                            maxIndex = j;
-                        }
-                    }
-                    var actionSize = discreteBuffer.Length;
-                    discreteBuffer[0] = maxIndex;
-                }
-                agentIndex++;
-            }
-        }
-    }
-
-    internal class ContinuousFromDiscreteOutputApplier : TensorApplier.IApplier
-    {
-        readonly ActionSpec m_ActionSpec;
-        int m_NumDiscretization;
-
-
-        public ContinuousFromDiscreteOutputApplier(ActionSpec actionSpec, int seed, ITensorAllocator allocator, int numDiscretization)
```

Comment:
this is no needed?  I made this for apply discrete output to continuous action for 3dball